agora_analytics/manifest.json

```diff
@@ -0,0 +1,57 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "hse7fl02-376a-33d0-7lpf-18s3r1284632",
+  "app_id": "agora-analytics",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "View Agora Analytics Collector data in Metrics Explorer",
+    "title": "The Agora Analytics",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Agora Analytics: Datadog Metrics Explorer Online User Count",
+        "image_url": "images/agora_rtc_online_user_count.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Agora Analytics: Datadog Metrics Explorer Audio Freeze Rate",
+        "image_url": "images/agora_rtc_audio_freeze_rate.png"
+      }
+    ],
+    "classifier_tags": [
+      "Category::Monitoring",
+      "Offering::Integration",
+      "Supported OS::Linux",
+      "Supported OS::Mac OS",
+      "Supported OS::Windows"
```

Comment:
Feel free to also add other categories, you can find the list of categories at the word cloud at the top of [this page](https://docs.datadoghq.com/integrations/).

Reply:
ok thx